code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -63,6 +131,23 @@
 	text_dehack = "[name]'s software has been reset!"
 	text_dehack_fail = "[name] does not seem to respond to your repair code!"
 
+/mob/living/simple_animal/bot/cleanbot/Crossed(atom/movable/AM)
+	zone_selected = pick(BODY_ZONE_L_LEG, BODY_ZONE_R_LEG)
+	if(has_gravity() && ismob(AM) && weapon)
+		var/mob/living/carbon/C = AM
+		if(!istype(C) || !C)
+			return
+		weapon.attack(C, src)
+		C.Knockdown(20)
+		var/mob/living/carbon/human/H = C
```

Comment:
I don't think you need to typecast to human here, all mobs have a mind datum.

Reply:
Cheers